tools/src/main/java/org/apache/kafka/tools/MetadataQuorumCommand.java

```diff
@@ -79,6 +82,10 @@ static void execute(String... args) throws Exception {
             .addArgument("--command-config")
             .type(Arguments.fileType())
             .help("Property file containing configs to be passed to Admin Client.");
+        parser
```

Comment:
this should probably go inside `addDescribeParser` since we want to parse for this only when --describe is called